posthog/api/plugin.py

```diff
@@ -175,7 +175,7 @@ def get_queryset(self):
         return queryset.none()
 
     def filter_queryset_by_parents_lookups(self, queryset):
-        parents_query_dict = self.get_parents_query_dict()
+        parents_query_dict = self.parents_query_dict
```

Comment:
does this allocation exist for a reason or could it be inlined?

Reply:
Definitely can be inlined now!